### ISSUE TITLE
Is it possible to visualize a Qiime 1.9 OTU biom file

### ISSUE THREAD

**Issue by wjacobus** [marbl/Krona#36]
I've used qiime to generate an OTU table for my datasets. Is it possible to viz with Krona

**Comment by ondovb** [comment]
BIOM format is not officially supported, but it seems like you could use ktImportText after converting to TSV (I have not tried this):
http://biom-format.org/documentation/biom_conversion.html

**Comment by ondovb** [comment]
It would be good to support BIOM officially.

**Comment by darnassiano** [comment]
Nope, I've converted it to tsv with biom, but had no results after :(

`biom convert -i taxa_summary/filtered_otu_cc_greengenes_L7.biom --table-type="OTU table" --to-tsv -o L7.txt`

**Comment by darnassiano** [comment]
Although, biom converted table **can be** visualized after adapting the data in the Excel Template.

**Comment by GenomicaMicrob** [comment]
Hi, I have created a little script to get all krona charts from an OTU table, check it here https://github.com/GenomicaMicrob/OTUsamples2krona
Browse to the end of the README file, there is solution that might help.
